spot_driver/src/interfaces/rclcpp_parameter_interface.cpp

```diff
@@ -9,7 +9,7 @@ constexpr auto kEnvVarNameAddress = "SPOT_IP";
 constexpr auto kEnvVarNameUsername = "BOSDYN_CLIENT_USERNAME";
 constexpr auto kEnvVarNamePassword = "BOSDYN_CLIENT_PASSWORD";
 
-constexpr auto kParameterNameAddress = "address";
+constexpr auto kParameterNameAddress = "hostname";
```

Comment:
do we want to change this variable too?  `kParameterNameAddress` --> `kParameterNameHostname`